docs/migration/2211_35/modernize-apps-migrated-from-6.8-to-2211.19.md

```diff
@@ -0,0 +1,430 @@
+
+```diff
+       "architect": {
+-        "server": {
+-          "builder": "@angular-devkit/build-angular:server",
+-          "options": {
+-            "outputPath": "dist/YOUR-APP-NAME/server",
+-            "main": "server.ts",
+-            "tsConfig": "tsconfig.server.json",
+-            "stylePreprocessorOptions": {
+-              "includePaths": [
+-                "node_modules/"
+-              ]
+-            },
+-            "inlineStyleLanguage": "scss"
+-          },
+-          "configurations": {
+-            "production": {
+-              "outputHashing": "media"
+-            },
+-            "development": {
+-              "optimization": false,
+-              "sourceMap": true,
+-              "extractLicenses": false,
+-              "vendorChunk": true,
+-              "buildOptimizer": false
+-            }
+-          },
+-          "defaultConfiguration": "production"
+-        },
+-        "serve-ssr": {
+-          "builder": "@angular-devkit/build-angular:ssr-dev-server",
+-          "configurations": {
+-            "development": {
+-              "browserTarget": "YOUR-APP-NAME:build:development",
+-              "serverTarget": "YOUR-APP-NAME:server:development"
+-            },
+-            "production": {
+-              "browserTarget": "YOUR-APP-NAME:build:production",
+-              "serverTarget": "YOUR-APP-NAME:server:production"
+-            }
+-          },
+-          "defaultConfiguration": "development"
+-        },
+-        "prerender": {
+-          "builder": "@angular-devkit/build-angular:prerender",
+-          "options": {
+-            "routes": [
+-              "/"
+-            ]
+-          },
+-          "configurations": {
+-            "production": {
+-              "browserTarget": "YOUR-APP-NAME:build:production",
+-              "serverTarget": "YOUR-APP-NAME:server:production"
+-            },
+-            "development": {
+-              "browserTarget": "YOUR-APP-NAME:build:development",
+-              "serverTarget": "YOUR-APP-NAME:server:development"
+-            }
+-          },
+-          "defaultConfiguration": "production"
+-        }
+```
+
+### `package.json`
+
+Please change the following `"scripts"` properties (because the new `application` builder handles the SSR and prerendering with different commands):
+
+1. Please remove properties `"dev:ssr"` and `"prerender"`
+
+```diff
+   "scripts": {
+-    "dev:ssr": "ng run YOUR-APP-NAME:serve-ssr",
+-    "prerender": "ng run YOUR-APP-NAME:prerender"
+```
+
+2. Please change value of the property `"build:ssr"` to `"ng build"`
+
+```diff
+   "scripts": {
+-     "build:ssr": "ng build && ng run YOUR-APP-NAME:server",
++     "build:ssr": "ng build"
+```
+
+3. Please change the value of the property `"serve:ssr"` to `"node dist/YOUR-APP-NAME/server/server.mjs"`
+
+```diff
+   "scripts": {
+-    "serve:ssr": "node dist/YOUR-APP-NAME/server/main.js",
++    "serve:ssr": "node dist/YOUR-APP-NAME/server/server.mjs",
```

Comment:
`MINOR:` Fresh apps with Angular >= 17 contains app's name in the script's name, e.g `"serve:ssr:YOUR-APP-NAME"`. If we want to be as close as the code generated by the latest Angular, we can consider this change.

Reply:
```suggestion
+    "serve:ssr:YOUR-APP-NAME": "node dist/YOUR-APP-NAME/server/server.mjs",
```